etc/module.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="../../../../lib/internal/Magento/Framework/Module/etc/module.xsd">
-    <module name="VinaiKopp_EavOptionSetup" setup_version="0.1.0">
+    <module name="VinaiKopp_EavOptionSetup" setup_version="1.0.0">
```

Comment:
Since composer uses git tags to determine the available versions I'll just add a git tag `0.1.0` when the merge is complete. You should be able to target that using that. If you object to using `0.1.0` since we are adding changes, we can use `0.2.0` if you prefer.